### PR TITLE
C

### PR DESCRIPTION
**Pull request by mrshayo**

### CODE REVIEW

File: fileF [hunk 1]
```diff
@@ -1 +1,3 @@
-This is a file in feature branch
```
Comment (jordanhildebrand2012) [inline fileF:1]: Are you sure you have done correctly, if so I will approve it. Great job

**Review by jordanhildebrand2012 — CHANGES_REQUESTED** [review]
Please fix the typo. I think will be best if you fix it. I'm 100% sure that's typo

File: fileF (hunk 1 above)
Comment (mrshayo) [reply fileF:1]: I have changed it. All fixed

**Review by jordanhildebrand2012 — APPROVED** [review]
Nice

**Pull request merged**